<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>products_Add</class>
 <widget class="QMainWindow" name="products_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>336</width>
         <height>302</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="getName">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Назва</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="getDesc">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Опис</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="getQuant">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="category_combo">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;</string>
        </property>
        <property name="text">
         <string>ВВЕДІТЬ ДАНІ ПРО ПРОДУКТ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="getPrice">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushData">
        <property name="geometry">
         <rect>
          <x>242</x>
          <y>270</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
       padding: 2px;
      border-radius: 3px;
       line-height: 1.25;
       background-color: #FC6E51;
       color: white;
       font-size: 12px;
       text-transform: uppercase;

    }
    QPushButton:hover {
     background-color: #FF5A36;
    }</string>
        </property>
        <property name="text">
         <string>ДОДАТИ</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="choseC">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Вибрати категорію</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="addC">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Додати нову</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QLineEdit" name="getCategoryName">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Назва</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="getCategoryDesc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Опис</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendC">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string> Додати категорію</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/prefix/img/gal.png</normaloff>:/prefix/img/gal.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="label_price">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Вартість</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_quant">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Кількість</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
